core/controllers/base_test.py

```diff
@@ -186,6 +181,13 @@ def test_that_no_get_results_in_500_error(self):
             if url == '/console_errors':
                 continue
 
+            if 'bulk_email_webhook_endpoint' in url:
```

Comment:
Can you add a code comment explain why we're doing this?

Reply:
Refactored.